### PR TITLE
Upgrade aarch64 port to Icedtea3.11.

### PR DESCRIPTION
**Pull request by phohensee**
### Description
Upgrade aarch64 port to Icedtea3.11.

### Motivation and context
Keep current with Icedtea aarch64 port.

### How has this been tested?
Hotspot jtreg tests pass better than x64 port.

### Platform information
    Works on OS: Amazon Linux 2 only
    Applies to version Corretto-8.202.08.3

### Contribution confirmation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.hpp [hunk 1]
```diff
@@ -24,8 +24,8 @@
  *
  */
 
-#ifndef CPU_X86_VM_C1_LIRASSEMBLER_X86_HPP
-#define CPU_X86_VM_C1_LIRASSEMBLER_X86_HPP
+#ifndef CPU_X86_VM_C1_LIRASSEMBLER_AARCH64_HPP
+#define CPU_X86_VM_C1_LIRASSEMBLER_AARCH64_HPP
```
Comment (navyxliu) [inline src/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.hpp]: CPU is still x86, which doesn't make sense.

File: src/hotspot/src/share/vm/runtime/globals.hpp [hunk 2]
```diff
@@ -4000,6 +4000,9 @@ class CommandLineFlags {
   product(bool, UseLockedTracing, false,                                    \
           "Use locked-tracing when doing event-based tracing")              \
                                                                             \
+  diagnostic(bool, UseUnalignedAccesses, false,                             \
```
Comment (navyxliu) [inline src/hotspot/src/share/vm/runtime/globals.hpp]: this flag is suspicious 
i) `AvoidUnalignedAccesses` above seems to play same role. 
i) hotspot doesn't have code to read this variable.

File: src/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.hpp (hunk 1 above)
Comment (phohensee) [reply src/hotspot/src/cpu/aarch64/vm/c1_LIRAssembler_aarch64.hpp]: Good catch, will fix.

File: src/hotspot/src/share/vm/runtime/globals.hpp (hunk 2 above)
Comment (phohensee) [reply src/hotspot/src/share/vm/runtime/globals.hpp]: Will delete UseUnalignedAccesses.

**Comment by phohensee** [comment]
Updated pull request, please re-review.

**Pull request merged**